packages/components/descriptions/src/descriptions-cell.ts

```diff
@@ -56,13 +57,13 @@ export default defineComponent({
               'el-descriptions__cell',
               'el-descriptions__label',
               {
-                'is-bordered-label': this.descriptions.border,
-                'is-vertical-label': this.descriptions.direction === 'vertical',
+                'is-bordered-label': border,
+                'is-vertical-label': direction === 'vertical',
               },
               labelAlign,
               labelClassName,
             ],
-            colSpan: this.descriptions.direction === 'vertical' ? span : 1,
+            colSpan: direction === 'vertical' ? span : 1,
```

Comment:
One more thing: we have multiple `direction === 'vertical'` here in this file, we should make it variable:

```ts
const isVertical = direction === 'vertical' 
```